cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -760,7 +760,7 @@ impl Inst {
                 format!("##fixed 4-size nop")
             }
             &Inst::Label { imm } => {
-                format!("##label=L{imm}")
+                format!("")
```

Comment:
Why we want to remove this?

Reply:
Well, this is largely a result of me starting to hack away at making vcode look more like zkasm, but before I got in too deep I realized it would be better to keep those as separate changesets. However since this one has already been made, it seemed unnecessary to revert this specific change.

In summary: no good reason to remove it *now*, but no good reason to *not* remove it *now* either.